app/Transformers/BibleTransformer.php

```diff
@@ -287,12 +287,9 @@ public function transformForV4($bible)
                     'filesets'     => $bible->filesets->mapToGroups(function ($item) {
                         return [$item['asset_id'] => $this->filesetWithMeta($item)];
                     }),
+                    'fonts' => $fonts
```

Comment:
Executing the postman tests I realized that a test is expecting that the fonts property is always set despite it is null.